backend/src/libs/packages/server-application/libs/helpers/create-error-response/handlers/handle-auth-error-response.ts

```diff
@@ -0,0 +1,28 @@
+import { ServerErrorType } from '#libs/enums/enums.js';
+import { AuthError } from '#libs/exceptions/exceptions.js';
+import { HTTPCode } from '#libs/packages/http/http.js';
+
+import { errorInfoLabelMap } from '../../error-info-label-map/error-info-label-map.js';
+import { type ErrorHandler } from '../types/types.js';
+
+const handleAuthErrorResponse: ErrorHandler = (error) => {
```

Comment:
We have discussed with other coaches and decided that we need a bit another approach here
1. rename to getAuthErrorInfo
2. ErrorHandler type and create type like ErrorInfo (our return type)
3. move this checks like `if (error instanceof AuthError) {` to getErrorInfo (old name - createErrorResponse), in getAuthErrorInfo we need to pass only AuthError and detect whether to call this handler on level above
4. rename info property to internalMessage
5. keep this helpers on the same level with getErrorInfo without additional folder, but export from helpers only getErrorInfo

And same for other handlers

Reply:
let me clarify. Should it look like this?
```ts
// getErrorInfo
if (error instanceof AuthError) {
  return getAuthErrorInfo(error);
}

if (error instanceof HTTPError) {
  return getHTTPErrorInfo(error);
}

return getDefaultErrorInfo(error);
```

Reply:
Yes)